flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["i386","arm","aarch64"]
```

Comment:
```suggestion
  "only-arches": ["x86_64"]
```

Reply:
you are right. specifying only-arches is more appropiate

Reply:
fixed problem in flathub.json in  [commit 41853e1054ed95d52dea0a472ec061afab5353e4](https://github.com/flathub/flathub/pull/4305/commits/41853e1054ed95d52dea0a472ec061afab5353e4)